### PR TITLE
ATO-NONE: Tidy code

### PR DESCRIPTION
**Pull request by hannah-leong**
## What?

Slight tidying of the code to make SonarLint happier

## Why?

Please include reason for the change and any other relevant context.

## Related PRs

Please include links to PRs in other repositories relevant to this PR.
Delete this section if not needed.

### CODE REVIEW

File: oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/AccessTokenService.java
Comment (ethanmills) [inline oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/AccessTokenService.java:172]: Why this change?

Comment (hannah-leong) [reply oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/AccessTokenService.java:172]: SonarLint doesn't like it due to unclear intentionality:
`Returning null instead of an actual array, collection or map forces callers of the method to explicitly test for nullity, making them more complex and less readable.
Moreover, in many cases, null is used as a synonym for empty.`

Comment (hannah-leong) [reply oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/AccessTokenService.java:171]: Am putting this back to null 👍

File: orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/KmsConnectionService.java
Comment (ethanmills) [inline orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/KmsConnectionService.java:33]: Does this work? Won't work where we're passing in an exception
```suggestion
            LOG.info("Localstack endpoint URI is present: {}", localstackEndpointUri.get());
```

**Review by ethanmills — APPROVED** [review]
This looks good, thanks for going through these. Please wait until the morning to merge.

**Review by ethanmills — APPROVED** [review]

**Pull request merged**
